.github/workflows/nodejs.yml

```diff
@@ -35,7 +36,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
For now, I would keep the node versions the same - they can be adjusted in another PR.